### PR TITLE
As a NH user I like to know which timeframe has been used as a default filter for the wire search

### PR DESCRIPTION
**Pull request by dzonidoo**
NHUB-560

### Purpose

- As a NH user I like to know which timeframe has been used as a default filter for the wire search

### What has changed

- A new Popover component has been created, used for displaying information about default timeframe

### Steps to test

1. Go to the Wire section
2. Search for some title
- Below the search results, you will see an icon for the popover.

### Checklist

- [x] This pull request is not adding new forms that use redux
- [x] This pull request is adding missing TypeScript types to modified code segments where it's easy to do so with confidence
- [x] This pull request is replacing `lodash.get` with optional chaining for modified code segments

Resolves: #[issue-number]

### CODE REVIEW

**Comment by thecalcc** [comment]
This Popover component used here you can't reuse the one from UIF/client-core right?

**Comment by tomaskikutis** [comment]
@dzonidoo pull request template is there for a reason - do fill it

File: assets/ui/components/Popover.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    component: React.ComponentType<{closePopup(): void}>;
+}
+
+interface IState {
+    isOpen: boolean;
+}
+
+export class Popup extends React.PureComponent<IProps, IState> {
+    elem: any;
+    referenceElem: any;
```
Comment (tomaskikutis) [inline assets/ui/components/Popover.tsx]: do not use `any` types

File: assets/search/components/SearchResultsBar/index.tsx [hunk 2]
```diff
@@ -196,13 +201,61 @@ class SearchResultsBarComponent extends React.Component<IProps, IState> {
...
+                                        }
+                                    </div>
+                                    {(() => {
+                                        if (!(this.props.activeDateFilter?.date_filter == null && defaultDateFilter?.name != null)) {
+                                            return null;
+                                        }
+                                        const dateFilterName = defaultDateFilter.name;
+                                        return (
+                                            <span className='d-flex align-items-center'>
+                                                <span className='popup-text--label'>{gettext('{{dateFilterName}}', {dateFilterName: dateFilterName})}</span>
```
Comment (tomaskikutis) [inline assets/search/components/SearchResultsBar/index.tsx]: placeholders are not translatable

**Review by tomaskikutis — COMMENTED** [review]
In ticket requirements it says this should be configurable. I don't any configuration code here. Did you do it?

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**